queries/css/indents.scm

```diff
@@ -3,8 +3,7 @@
   (declaration)
 ] @indent
 
-[
-  "}"
-] @branch
+(block ("}") @branch)
+("}") @dedent
```

Comment:
those parens in the two lines are not necessary. but they neither harm.

Reply:
Ah, interesting. I'm not really familiar with the TS Scheme language (or Scheme in general). Are you saying this could be written as

```scheme
block "}" @branch
"}" @dedent
```
?

Reply:
Only the parentheses around `"}"`. The DSL has nothing to do with real scheme